include/tscore/ParseRules.h

```diff
@@ -33,7 +33,7 @@ typedef unsigned int CTypeResult;
 
 // Set this to 0 to disable SI
 // decimal multipliers
-#define USE_SI_MULTILIERS 1
+#define USE_SI_MULTIPLIERS 1
 
```

Comment:
Please fix the ParseRules.cc(in ink_atoui64) as well .

Reply:
good catch. I probably shouldn't have split this change across multiple PRs